.github/workflows/valid-links.yml

```
@@ -4,18 +4,33 @@ on:
   push:
   pull_request:
   schedule:
-    - cron: '0 3 * * *'
+    - cron: "0 3 * * *"
 
 jobs:
   verify-links:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v1
+      - uses: actions/checkout@master
```

```suggestion
      - uses: actions/checkout@v2
```

Using `@master` is usually not recommended unless you need features only available on master.

---

Good catch!